### ISSUE TITLE
Question about drawers oredict auto-conversion

### ISSUE THREAD

**Issue by Adaptivity** [jaquadro/StorageDrawers#120]
This is not an issue this is rather a question about oredict auto-conversion. I'm playing first time with this mod and I don't know how to convert ore dictionary compatible stuff. I tried to put copper ore (with the same ore dictionary name) from different mods in the same drawer but when I have put copper ore from one mod I can't put ore from another. Also, tried the same with ingots. I have *Item Conversion* option enabled both on client and server. Maybe I'm doing something wrong? I also tried to find some information about it but I failed. Sorry for posting this there but this is faster way to contact you for me. It can be closed when you answer me.
Version of mod: 1.5.11
Forge: 1481

**Comment by jaquadro** [comment]
There was a problem with a few of the nuggets due to TCon ore berries, which I've added to whitelist in 1.5.12.

There's a few reasons why ore blocks will probably not convert, and it's related to the tighter conversion restrictions that were added a few versions ago.  Considering the breadth of ore blocks and how they all get registered, I think it would be a bad idea to support this conversion anyway.

Nuggets/Ingots/Blocks should generally work, but depending on your mod set, it's possible some of the ore dictionary keys will get "shut down" because of ambiguities.  For example, the mod can't tell the difference between "gold coins are gold nuggets are gold ore berries" and "iron ingots are metal ingots are gold ingots", the first being a safe conversion chain and the second being unsafe.

You will need to provide a dump of your ore dictionary for me to be able to tell.  If you install the MineTweaker mod, a dump can be made with the command "/mt oredict"
